Update native logging docs (#5051)
mendix/docs: content/en/docs/refguide/mobile/distributing-mobile-apps/logging.md

{{% alert color="warning" %}}
Please note the following current limitations regarding native client logs:

> * Crash logs are not supported currently

Reviewer: I think this is partially true - we do log client crashes, and they are sent to the runtime with the logs. But we don't log native crashes, which I believe happen less.

Author: Corrected!

The following log nodes are used by Mendix when writing log messages:

{{% alert color="warning" %}}

> From Mendix Studio Pro 9.18.1 and above log messages are filtered based on the log levels set in the Cloud Portal app.

Reviewer: Should we add a warning that enabling "trace" may cause too many logs to be sent to the runtime server, so it should not be kept on for longer time frames?

Author: Added!

## 5 Sending Log Messages To Runtime {#sending-client-log-nodes-to-runtime}

> The native client stores log messages locally on the device. When **Enable sending logs to runtime** is selected, the native client will attempt to send log messages whenever a quota of 1,000 messages has been reached or after 1 hour from the last sending attempt.

Reviewer: ```suggestion
The native client stores logs on the device's memory. When **Enable sending logs to runtime** is selected, the native client will attempt to send logs whenever a quota of 1,000 logs has been reached or after 1 hour from the last sending attempt.
```

> When the app goes to background, the native client will save buffered log message on the file system and will restore them once the app is active again. This is in case log messages reach the limit but there is no network connectivity when the native client would attempt to send these log messages. 

Reviewer: ```suggestion
When the app goes to the background or closes, the native client will save buffered log messages on the file system and will restore them once the app is active again.
```

The following sentence below is not clear (to me). What do we want to explain here?

>  This is in case of log messages reach the limit but there is no network connectivity when the native client would attempt to send these log messages.

> In this scenario, it will discard older messages so that it will not overflow with too many logs. An additional `info` log message will be added with the number of discarded logs and the timestamps of when these log messages were ignored.

Reviewer: Should we also mention the format of the logs, such as the unique session id and the date/time of the log as the date in the cloud portal is the date of the collection, not the date of the log itself.

Author: Done! ✅
